websocket-test.html

```
@@ -0,0 +1,278 @@
+        </div>
+        
+        <div>
+            <button id="connectBtn" onclick="connect()">Connect</button>
+            <button id="disconnectBtn" onclick="disconnect()" disabled>Disconnect</button>
+            <button onclick="clearLog()">Clear Log</button>
+            <button onclick="testSensorData()">Send Test Sensor Data</button>
+        </div>
+        
+        <div class="input-group">
+            <label for="messageInput">Send Custom Message (JSON):</label>
+            <textarea id="messageInput" placeholder='{"type": "test", "message": "Hello WebSocket!"}'></textarea>
```

HTMLHint (1.5.0)

[error] The value of attribute [ placeholder ] must be in double quotes.

(attr-value-double-quotes)